.fixtures.yml

```diff
@@ -0,0 +1,9 @@
+fixtures:
+  symlinks:
+    minecraft: "#{source_dir}"
+  forge_modules:
+    stdlib: 'puppetlabs/stdlib'
+    java: 'puppetlabs/java'
+    wget: 'maestrodev/wget'
+  repositories:
+    archive: "https://github.com/voxpupuli/puppet-archive.git"
```

Comment:
@rnelson0 @igalic what do you think about getting the module from the forge? I think we mostly use githubs HEAD directly instead of the last release? I would prefer that so we can catch issues on their master branches easier/before a release.